01-getting-started/003-ecr-setup.md

```diff
@@ -10,8 +10,41 @@ This guide will guide you through the creation of an ECR (Elastic Container Regi
 
 AWS resources are provisioned through the [cloud-platform-environments](https://github.com/ministryofjustice/cloud-platform-environments/) repository, per environment. Your application might be using multiple environments, however, you only need one image repository so we will be working in only one of the environments (it doesn't matter which one).
 
-## Write the terraform configuration
-In your local clone of the environments repository, `cd` to your environment's path and create a new directory called `resources`. Inside `resources`, create a new file called `ecr.tf` with the following contents:
+### Creating the registry
+
+1\. In order to create the ECR Docker registry, git clone the repo and create a new branch.
+
+```bash
+
+  $ git clone git@github.com:ministryofjustice/cloud-platform-environments.git #git clone repo
+
+  $ cd cloud-platform-environments # navigate into cloud-platform-environments directory.
+
+  $ git co -b add_ecr   # create and checkout new branch.
+
+```
+
+2\. You will need to navigate to your service's directory which is located in the namespaces directory. Create a directory named "resources".
+
+```bash
+
+  $ cd namespaces/$your_service  #navigate to your service's directory.
+
+  $ mkdir resources # make directory called resources
+
+  $ cd namespaces/$your_service/resources
```

Comment:
I think we are moving to the convention of angle brackets for variables in docs because the dollar sign sometimes confuses things (i.e. people think it should be left in). So can we make this `namespaces/<your-service>` etc?

Reply:
<> break the parser, and if I escape when doing a raw view to copy/paste it will have &lt, annoying